api/users/serializers.py

```diff
@@ -509,6 +509,8 @@ def create(self, validated_data):
                 send_confirm_email(user, email=address)
         except ValidationError as e:
             raise exceptions.ValidationError(e.args[0])
+        except BlacklistedEmailError as e:
```

Comment:
Super small nitpick, but you're not using the `e` variable so it can be taken out

Reply:
gah, I forgot to remove that, I was originally reraising the Invalid Email error -thanks